cli/cmd/auth.go

```diff
@@ -138,14 +138,17 @@ func verifyAuthParams(authParams *authCmdParams) error {
 	return nil
 }
 
-func lookupHostname(hostname string) bool {
+type resolveFunc func(string) ([]string, error)
+type sleepFunc func(time.Duration)
+
+func lookupHostname(hostname string, lookupFn resolveFunc, sleepFn sleepFunc) bool {
```

Comment:
I'm guessing this improves testability?

Reply:
yes, the unit test took about 20 seconds, as it was directly hitting `net.LookupHost` as well as `time.Sleep`